### PR TITLE
feature(ycsb): [dynamodb] add support for TTL

### PR DESCRIPTION
**Pull request by fruch**
creating ycsb image that includes:
https://github.com/scylladb/YCSB/pull/1

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by fgelcer — DISMISSED** [review]
LGTM

**Review by yarongilor — DISMISSED** [review]
@fruch shell i test it in a regression? or is it ready to be tested in a draft PR using TTL?
LGTM

File: docker/ycsb/Dockerfile [hunk 1]
```diff
@@ -16,6 +16,9 @@ RUN cd /YCSB/scylla/target && mkdir -p YCSB && tar xvvf ycsb-scylla-*.tar.gz -C
 
 
 FROM openjdk:8 as app
+
+RUN apt-get update && apt-get install -y python3
```
Comment (vponomaryov) [inline docker/ycsb/Dockerfile:20]: It will make the final image be bigger than it can be if we add 'python3' installation to the lines 3-5

Comment (fruch) [reply docker/ycsb/Dockerfile:20]: we need that in run time, not only in build time.

Comment (vponomaryov) [reply docker/ycsb/Dockerfile:20]: `RUN` command is used only in the build stage and the probability that some new update appears after lines 3-5 is about `0`. Also, `python3` version won't be updated that fast too. So, I don't see any relation of this new line to the statement about the `runtime` and my original concern is still valid.

Comment (fruch) [reply docker/ycsb/Dockerfile:20]: this is a multi stage build, put the python3 install on the first stage isn't going to help, it won't be available on the 2nd stage, nor in the final image.

I don't understand what are you commenting about

Comment (vponomaryov) [reply docker/ycsb/Dockerfile:20]: Oh, I mislooked that we are in the different layer. No concern then.

**Review by vponomaryov — APPROVED** [review]

**Pull request merged**